src/main/java/com/enderio/base/data/recipe/MaterialRecipeProvider.java

```diff
@@ -229,7 +229,7 @@ private void addCrystals(Consumer<FinishedRecipe> recipeConsumer) {
             .pattern("PPP")
             .pattern("PDP")
             .pattern("PPP")
-            .define('P', EIOTags.Items.INGOTS_PULSATING_ALLOY)
+            .define('P', EIOItems.PULSATING_ALLOY_NUGGET)
```

Comment:
Can we use the nuggets tag here please?

Reply:
Oh did we have a tag for that?

Reply:
Yeah ingot and nugget tags are a recent addition :) https://github.com/Team-EnderIO/EnderIO/blob/dev/1.20.1/src/main/java/com/enderio/base/common/tag/EIOTags.java#L64-L82